src/scripts.js

```diff
@@ -14,99 +14,125 @@ import Hydration from './Hydration';
+  if (Object.values(a)[0] > Object.values(b)[0]) {
+    return -1;
+  }
+  if (Object.values(a)[0] < Object.values(b)[0]) {
+    return 1;
+  }
+  return 0;
+});
+
+//querySelectors
+
+// let dropdownEmail = document.querySelector('#dropdown-email');
+// let dropdownFriendsStepsContainer = document.querySelector('#dropdown-friends-steps-container');
+// let dropdownGoal = document.querySelector('#dropdown-goal');
+// let dropdownName = document.querySelector('#dropdown-name');
+// let stepsCalendarTotalActiveMinutesWeekly = document.querySelector('#steps-calendar-total-active-minutes-weekly');
+// let stepsCalendarTotalStepsWeekly = document.querySelector('#steps-calendar-total-steps-weekly');
+// let stepsFriendAverageStepGoal = document.querySelector('#steps-friend-average-step-goal');
+// let stepsInfoActiveMinutesToday = document.querySelector('#steps-info-active-minutes-today');
+// let stepsInfoMilesWalkedToday = document.querySelector('#steps-info-miles-walked-today');
+// let stepsFriendActiveMinutesAverageToday = document.querySelector('#steps-friend-active-minutes-average-today');
+// let stepsFriendStepsAverageToday = document.querySelector('#steps-friend-steps-average-today');
+// let stepsTrendingButton = document.querySelector('.steps-trending-button');
+// let trendingStepsPhraseContainer = document.querySelector('.trending-steps-phrase-container');
+//let dailyOz = document.querySelectorAll('.daily-oz');
+// let hydrationFriendOuncesToday = document.querySelector('#hydration-friend-ounces-today');
+//let hydrationInfoGlassesToday = document.querySelector('#hydration-info-glasses-today');
+// let sleepCalendarHoursAverageWeekly = document.querySelector('#sleep-calendar-hours-average-weekly');
+// let sleepCalendarQualityAverageWeekly = document.querySelector('#sleep-calendar-quality-average-weekly');
+// let sleepFriendLongestSleeper = document.querySelector('#sleep-friend-longest-sleeper');
+//let sleepFriendWorstSleeper = document.querySelector('#sleep-friend-worst-sleeper');
+// let sleepInfoHoursAverageAlltime = document.querySelector('#sleep-info-hours-average-alltime');
+// let sleepInfoQualityAverageAlltime = document.querySelector('#sleep-info-quality-average-alltime');
+// let sleepInfoQualityToday = document.querySelector('#sleep-info-quality-today');
+// let stairsCalendarFlightsAverageWeekly = document.querySelector('#stairs-calendar-flights-average-weekly');
+// let stairsCalendarStairsAverageWeekly = document.querySelector('#stairs-calendar-stairs-average-weekly');
+//let stairsFriendFlightsAverageToday = document.querySelector('#stairs-friend-flights-average-today');
+//let stairsInfoFlightsToday = document.querySelector('#stairs-info-flights-today');
+//let stairsTrendingButton = document.querySelector('.stairs-trending-button');
+//let trendingStairsPhraseContainer = document.querySelector('.trending-stairs-phrase-container');
 let headerName = document.querySelector('#header-name');
```

Comment:
We can organize these querySelectors so that they're easier to find. We could also create a helper function that takes an argument of 'className' and creates a querySelector so that we can access the element inside of the related function without having to declare so many global variables.